layouts/shortcodes/add_biobank_form.html

```diff
@@ -0,0 +1,166 @@
+          <label class="form-check-label" for="yes">Yes</label>
+        </div>
+        <div class="form-check">
+          <input class="form-check-input" type="radio" name="biobank_sverige" id="no" value="FALSE">
+          <label class="form-check-label" for="no">No</label>
+        </div>
+        <small class="form-text">The biobank belong to the Biobank Sverige network.</small>
+      </div>
+      <div class="form-group mb-3">
+        <label for="ivo_regnum"><b>Biobank registration number at IVO:</b></label>
+        <input type="text" class="form-control" name="ivo_regnum" id="ivo_regnum" aria-describedby="ivoRegNumHelp">
+        <small id="ivoRegNumHelp" class="form-text">
+          If applicable, provide the registration number of the biobank at Inspektionen för vård och omsorg.
+        </small>
+      </div>      
+      <div class="form-group mb-3">
+        <label for="bbmri_id"><b>BBMRI-ERIC biobank ID:</b></label>
+        <input type="text" class="form-control" name="bbmri_id" id="bbmri_id" aria-describedby="bbmriIdHelp">
+        <small id="bbmriIdHelp" class="form-text">
+          If the biobank is already listed in BBMRI-ERIC, please provide its ID. These IDs are based on MIABIS 2.0 standard (ISO 3166-1 alpha-2 + underscore + biobank national ID or name), prefixed with bbmri-eric:ID: string. For example, <i>bbmri-eric:ID:SE_9999</i>.
+        </small>
+      </div>      
+      <div class="form-group mb-3">
+        <label for="url"><b>Biobank homepage URL:</b></label>
+        <input type="url" class="form-control" name="url" id="url" aria-describedby="urlHelp">
+        <small id="urlHelp" class="form-text">
+          Biobank URL (as required by <a target="_blank" href="https://github.com/MIABIS/miabis/blob/master/Data-describing-Biobank.md">MIABIS-2.0-04</a>).
+        </small>
+      </div>      
+      <div class="form-group mb-3">
+        <label for="description"><b>Biobank description:</b></label>
+        <textarea class="form-control" id="description" rows="3" name="description" aria-describedby="descriptionHelp"></textarea>
+        <small id="descriptionHelp" class="form-text">
+          Biobank description (as required by <a target="_blank" href="https://github.com/MIABIS/miabis/blob/master/Data-describing-Biobank.md">MIABIS-2.0-08</a>).
+        </small>
+      </div>      
+      <div class="form-group mb-3">
+        <label for="logoURL"><b>Biobank logo URL:</b></label>
+        <input type="url" class="form-control" name="logoURL" id="logoURL" aria-describedby="logoURLHelp">
+        <small id="logoURLHelp" class="form-text">
+          If the biobank has a logotype, please provide a URL of this logo (http://..). The logo will be displayed on the page of the biobank in the registry.
+        </small>
+      </div>      
+    </div>
+  </div>
+  </div>
+  <div class="card">
+    <div class="card-header" role="tab" id="generalinfo">
```

Comment:
`pa11ly` is complaining about this. There another element having the same id `generalinfo` (in line 5). So can you change the id for this element.